projenrc/github-workflows.ts

```diff
@@ -147,15 +149,16 @@ export function buildMonthlyIssuesMetricsWorkflow(project: AwsCdkConstructLibrar
 export function buildAutoApproveWorkflow(project: AwsCdkConstructLibrary) {
   const autoapprove: Job = {
     runsOn: ['ubuntu-latest'],
-    if: "(github.event.pull_request.user.login == 'emerging-tech-cdk-constructs-bot' || github.event.pull_request.user.login == 'generative-ai-cdk-constructs-bot') && contains(github.event.pull_request.labels.*.name, 'auto-approve')",
+    if: "contains(github.event.pull_request.labels.*.name, 'auto-approve')",
```

Comment:
Checking on permissions for labeling

Reply:
Only people with [at least triage access](https://docs.github.com/en/enterprise-cloud@latest/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-a-label) can assign a label to a PR